prowler/providers/aws/services/ecr/ecr_service.py

```diff
@@ -141,6 +141,9 @@ def __get_image_details__(self, regional_client):
                                 if image is not None:
                                     severity_counts = None
                                     last_scan_status = None
+                                    artifact_media_type = image.get(
```

Comment:
This got my attention without having started the PR review. What if we just store the artifacts with media type `"application/vnd.docker.container.image.v1+json"`? We are not doing anything else with the rest of media types so  for the sake of memory storage and speed I think it'd be better just to store what we are going to analyze. Then in the future we can extend it.

Reply:
> What if we just store the artifacts with media type `"application/vnd.docker.container.image.v1+json"`?

I was not sure if prowler is using other types of artifacts but if prowler only cares about Docker images, then yes, the `ecr_service.py` could them sort out early already to save RAM and CPU cycles.

Reply:
It seems that detecting a scannable image is not as easy as I initially thought because some artifact media types are used for scannable and not scannable artifacts 😞 I have to dive deeper to solve this problem.
Furthermore, AWS introduced a new version of basic scanning which does not support imageScanFindingsSummary and imageScanStatus in the DescribeImages API. To view these, we have to use the DescribeImageScanFindings API. 
I'll adapt prowler's `ecr_servive.py` to take this change into account.
So maybe it's better to **review this PR a bit later** when I'm done with it.

Reply:
That's totally fine! Thank you very much for your work on this it an absolutely pleasure to work with you 🚀 

I'm going to convert the PR to `draft` and once it will be ready for review you can mark it as `ready for review` again.

Thanks a lot!

Reply:
ready for review 😄

Reply:
Great, I'll see if @sergargar can review that today, if not we will review it on Friday. Thanks for your work Kay!